src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1404,49 +1567,8 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 				}
 
```

Comment:
refactored code for rendering display after a long press is entered to renderDisplayForMultiPadPress